pkg/forwarder/transaction.go

```diff
@@ -120,6 +167,7 @@ func (t *HTTPTransaction) Process(ctx context.Context, client *http.Client) erro
 	} else if resp.StatusCode > 400 {
 		t.ErrorCount++
 		transactionsErrors.Add(1)
+		transactionsHTTPErrors.Add(1)
```

Comment:
Shouldn't we also add one for `400`/`404`/`413`/`403` ?

Reply:
Also I think we should separate client and server side errors (4xx and 5xx)

Reply:
Should we add dedicated error types for 400/404/413/403 or just one for 4xx and one for 5xx ?

Reply:
I would say that one for client and one for server is enough but I may be wrong

Reply:
I was thinking about this and I don't think there is a clear winner between the two. Having a static set of expvar brings default values which makes it easier to build dashboard around them. That said it make us lose some granularity which is bad for troubleshooting.

Reply:
We should probably go with dedicated error types as we want as much granularity as possible for troubleshooting

Reply:
For each status code ? We can't do arrays in expvar, that would be ugly to write

Reply:
For every status code. I think it's fine to populate an expvar map dynamically in this particular case.

Reply:
Okay

Reply:
The only drawback is that it makes it harder to build dashboard since there won't be default values for this expvars so they won't show up until an error of this type actually appends.